drake/solvers/MathematicalProgram.cpp

```diff
@@ -106,9 +115,10 @@ class NonlinearProgram : public MathematicalProgram {
     MixedIntegerQuadraticProgram {};
     struct LinearProgram : public QuadraticProgram, public
     MixedIntegerLinearProgram {
-    virtual MathematicalProgramInterface* addLinearEqualityConstraint() { return new
+    virtual MathematicalProgramInterface* AddLinearEqualityConstraint() { return
+   new
     LinearProgram; };
```

Comment:
yup, that's clang-format's doing.  I'm not sure it's worth messing with since the formatting would presumably be different if it weren't in a comment anyway.